ui/components/MesherySettings.js

```diff
@@ -393,14 +393,14 @@ class MesherySettings extends React.Component {
                 tab="registry"
               />
             </Tooltip>
-            <Tooltip title="Credential" placement="top">
+            {/* <Tooltip title="Credential" placement="top">
```

Comment:
@MeenuyD the eslint check will fail due to trailing space, use `//` to comment out the code for now

Reply:
can you please explain it more?